tests/projects/test_app_project_api.py

```diff
@@ -89,10 +89,11 @@ def test_app_project_api_single_poll_module(
 @pytest.mark.django_db
 def test_app_project_serializer(project_factory, module_factory, phase_factory,
                                 apiclient):
+    html_string = '<p>text with a <strong>bold</strong> bit</p>'
```

Comment:
Even though, we do not remove the tags anymore, you could test that the strip works and removes whitespace.